accounting/accounting_cli.py

```diff
@@ -0,0 +1,148 @@
+        dest="view_user", type=str, help="username", metavar=("USER_NAME")
+    )
+
+    subparser_add_user = subparsers.add_parser(
+        "add-user", help="add a user to the accounting database"
+    )
+    subparser_add_user.add_argument(
+        "--user_name",
+        required=True,
+        help="username",
+        metavar="USER_NAME",
+        dest="add_user",
+        action="append",
+    )
+    subparser_add_user.add_argument(
+        "--admin_level",
+        required=True,
+        help="admin level",
+        metavar="ADMIN_LEVEL",
+        dest="add_user",
+        action="append",
+    )
+    subparser_add_user.add_argument(
+        "--account",
+        required=True,
+        help="account to charge jobs against",
+        metavar="ACCOUNT",
+        dest="add_user",
+        action="append",
+    )
+    subparser_add_user.add_argument(
+        "--parent_acct",
+        required=True,
+        help="parent account",
+        metavar="PARENT_ACCOUNT",
+        dest="add_user",
+        action="append",
+    )
+    subparser_add_user.add_argument(
+        "--shares",
+        required=True,
+        help="shares",
+        metavar="SHARES",
+        dest="add_user",
+        action="append",
+    )
+    subparser_add_user.add_argument(
+        "--max_jobs",
+        required=True,
+        help="max jobs",
+        metavar="MAX_JOBS",
+        dest="add_user",
+        action="append",
+    )
+    subparser_add_user.add_argument(
+        "--max_wall_pj",
+        required=True,
+        help="max wall time per job",
+        metavar="MAX_WALL_PJ",
+        dest="add_user",
+        action="append",
+    )
+
+    subparser_delete_user = subparsers.add_parser(
+        "delete-user", help="remove a user from the accounting database"
+    )
+    subparser_delete_user.add_argument(
+        dest="delete_user", type=str, help="username", metavar=("USER_NAME")
+    )
+
+    subparser_edit_user = subparsers.add_parser("edit-user", help="edit a user's value")
+    subparser_edit_user.add_argument(
+        "--user_name",
+        required=True,
+        help="username",
+        metavar="USER_NAME",
+        dest="edit_user",
+        action="append",
+    )
+    subparser_edit_user.add_argument(
+        "--field",
+        required=True,
+        help="column name",
+        metavar="FIELD",
+        dest="edit_user",
+        action="append",
+    )
+    subparser_edit_user.add_argument(
+        "--new_value",
+        required=True,
+        help="new value",
+        metavar="VALUE",
+        dest="edit_user",
+        action="append",
+    )
+
+    args = parser.parse_args()
+
+    if hasattr(args, "view_user"):
+        aclif.view_user(conn, args.view_user)
+    elif hasattr(args, "add_user"):
+        aclif.add_user(conn, args.add_user)
+    elif hasattr(args, "delete_user"):
+        aclif.delete_user(conn, args.delete_user)
+    elif hasattr(args, "edit_user"):
+        aclif.edit_user(conn, args.edit_user)
+
```

Comment:
I'd also add an else condition for the situation where no subcommand is provided:

```suggestion
    else:
        parser.print_usage()
```